README.md

```diff
@@ -68,12 +68,14 @@ programs.neovim = {
 ```
 
 ##### Windows
-> Coming soon.
-
+You can get sqlite3 precompiled binaries for Windows [here](https://www.sqlite.org/download.html).
+You'll be looking for `sqlite3.dll` instead of `libsqlite3.so`, be
```

Comment:
just paste in `let g:sql_clib_path =  path/to/sqlite3.dll` in this line. Everything else looks good

Reply:
Oh I guess I missed it

Reply:
Something like this ?
```
You can get sqlite3 precompiled binaries for Windows here. You'll be looking for sqlite3.dll instead of libsqlite3.so (let g:sql_clib_path = path/to/sqlite3.dll), be careful to also use / in the path you specify (Windows's paths usually use \).
```

Reply:
How about 
```
1. get sqlite3 precompiled binaries for Windows here.
2. set sql_clib_path to where `sqlite3.dl` is located, eg. `let g:sql_clib_path = path/to/sqlite3.dll`, use `/` instead of `\`
```

Reply:
Oh ok one sec